.github/workflows/build.yml

```diff
@@ -67,10 +59,76 @@ jobs:
             build/${{ matrix.version }}/report.json
             build/${{ matrix.version }}/progress-commit.json
 
+  verifyNonMatching:
+    # This is a *private* build container.
+    # See docs/github_actions.md for more information.
+    container: ghcr.io/bfbbdecomp/bfbb-build:main
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # Add game versions here
+        version: [GQPE78]
+
+    steps:
+      # Checkout the repository
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
```

Comment:
```suggestion
```
Only the previous report job needs this (so it can be removed from the main build job too)